common/lib/capa/capa/templates/choicegroup.html

```diff
@@ -1,43 +1,52 @@
-<form class="choicegroup capa_inputtype" id="inputtype_${id}">
+<form class="choicegroup capa_inputtype" id="${id}-form">
     <fieldset role="${input_type}group" aria-label="${label}">
```

Comment:
We can remove the role here (fieldset has an implied role of group so specifying it is redundant and can actually have negative consequences.) We can also remove the aria-label since the legend is the accessible name, or label, for the fieldset (or group).